recommendations.go

```diff
@@ -13,6 +13,17 @@ type Recommendations struct {
 	NumberNextRecommsCalls int `json:"numberNextRecommsCalls"`
 }
 
+// RecommendItemsToUser recommends top-N items that are most likely to be of high value for the given user.
+//
+// The most typical use cases are recommendations on the homepage, in some “Picked just for you” section, or in email.
```

Comment:
This is rather our use case

Reply:
Copied verbatim from the official documentation: https://docs.recombee.com/api.html#recommend-items-to-user

I'm fine with removing this paragraph as it may be too verbose for out purposes.

Reply:
Try to change it. This is official documentation, refer it as reference, instead of copying it